Index: src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Main.java b/src/reCap/Main.java
rename from src/Main.java
rename to src/reCap/Main.java
--- a/src/Main.java	(revision 46366549422384c244acd525a76478fb700b9df0)
+++ b/src/reCap/Main.java	(date 1663657549785)
@@ -1,3 +1,5 @@
+package reCap;
+
 import java.lang.reflect.Array;
 import java.util.Arrays;
 import java.util.Collections;
